mkdocs.yml

```diff
@@ -1,19 +1,30 @@
 site_name: Analog Devices MSDK Documentation
-site_author: Analog Devices
-copyright: 'Copyright (C) 2022-2023 Maxim Integrated Products, Inc. All Rights Reserved. (now owned by Analog Devices, Inc.), Copyright (C) 2023 Analog Devices, Inc. All Rights Reserved. This software is proprietary to Analog Devices, Inc. and its licensors.'
+site_author: Analog Devices, Inc.
+copyright: 'Copyright (C) 2022-2023 Maxim Integrated Products, Inc. All Rights Reserved. (now owned by Analog Devices, Inc.), Copyright (C) 2023-2024 Analog Devices, Inc. All Rights Reserved. This software is proprietary to Analog Devices, Inc. and its licensors.'
```

Comment:
The `This software is proprietary to Analog Devices, Inc. and its licensors.` line can be deleted. This line should have also been removed from most other files within the MSDK due to the permissive licensing of the MSDK.